scripts/battlefields/Throne_Room/kindred_spirits.lua

```diff
@@ -0,0 +1,101 @@
+-----------------------------------
+-- Area: Throne Room
+-- BCNM 60 - Kindred Spirits
+-- !pos -111 -6 0.1 165
+-----------------------------------
+local ID = require("scripts/zones/Throne_Room/IDs")
+require("scripts/globals/battlefield")
+require("scripts/globals/items")
+require("scripts/globals/zone")
+-----------------------------------
+
+local content = Battlefield:new({
+    zoneId        = xi.zone.THRONE_ROOM,
+    battlefieldId = xi.battlefield.id.KINDRED_SPIRITS,
+    maxPlayers    = 6,
+    levelCap      = 60,
+    timeLimit     = utils.minutes(30),
+    index         = 2,
+    entryNpc      = "_4l1",
+    exitNpcs      = { "_4l2", "_4l3", "_4l4" },
+    requiredItems = { xi.items.MOON_ORB, message = ID.text.A_CRACK_HAS_FORMED },
+
+})
+
+function content:isValidEntry(player, npc)
+    return self.entryNpc == '_4l1'
+end
+
+function content:onEntryEventUpdate(player, csid, option, npc)
+    if Battlefield.onEntryEventUpdate(self, player, csid, option, npc) then
+        if npc:getName() == '_4l1' then
+            self.exitLocation = 1
+            self.lossEventParams  = { [5] = 1 }
+        else
+            self.exitLocation = 0
+            self.lossEventParams  = {}
+        end
+    end
+end
+
+content.groups =
+{
+    {
+        mobs = { "Count_Andromalius" },
+    },
+    {
+        mobs = { "Duke_Amduscias" },
+    },
+    {
+        mobs = { "Grand_Marquis_Chomiel" },
+    },
+    {
+        mobs = { "Duke_Dantalian" },
+    },
+    {
+        mobs = { "Demons_Avatar" },
+    },
+    {
+        mobs = { "Demons_Elemental" },
+    },
+}
```

Comment:
Same as what I said in the When Hell Freezes Over. You do need a group for the Demons_Elemental since that isn't included in essential mobs.

Somewhat related but we like to but an extra space after multiline tables to help with readability. These aren't so bad since they're so short but an extra line helps a lot with larger tables. Single table entries like these you could collapse down into a single line. `{ mobs = { "Demons_Elemental" } }`

Reply:
updated, however kindred spirits is not spawning the chest, and as we discussed, that is likely because of how the ID's are laid out. Just wanted to put that reminder in there.

Reply:
Thanks, I remembered after commenting on this. I'm going to work on fixing that now.